From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Jake Bailey <5341706+jakebailey@users.noreply.github.com>
Date: Fri, 14 Oct 2022 15:49:39 -0700
Subject: [PATCH] Add gitlens settings suggestion

While GitHub automatically uses this file if present, GitLens in VS Code
does not. Add the right option to our example settings.json for those
who use the extension.

Unfortunately, you can't leave this enabled if you want to look at the
repo _without_ the file; git blame just crashes when the file isn't
present. I'm not sure that there's a workaround for that.
---
 .vscode/settings.template.json | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/.vscode/settings.template.json b/.vscode/settings.template.json
index ccfcfc2631..43d69a60b6 100644
--- a/.vscode/settings.template.json
+++ b/.vscode/settings.template.json
@@ -6,4 +6,10 @@
 
     // To use the locally built compiler, after 'npm run build':
     // "typescript.tsdk": "built/local"
+
+    // To ignore commits listed in .git-blame-ignore-revs in GitLens:
+    // "gitlens.advanced.blame.customArguments": [
+    //     "--ignore-revs-file",
+    //     ".git-blame-ignore-revs"
+    // ]
 }
-- 
2.38.1

